core/src/apps/polkadot/seed.py

```
@@ -39,7 +39,7 @@ def _derive_path(root: bip32.HDNode, path: Bip32Path) -> bip32.HDNode:
         node.derive_path(path)
         return node
 
-    def verify_path(self, path: Bip32Path) -> None:
+    def verify_path(self, path: Bip32Path, _force_strict: bool = False) -> None:
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)